src/storage.js

```diff
@@ -4,8 +4,11 @@ const { redis } = require('./lib/redis');
 const FEED_ID = 'feed_id';
 const FEEDS = 'feeds';
 
+const POST_ID = 'post_id';
+const POSTS = 'posts';
+
 module.exports = {
-  addFeed: async (name, url) => {
+  addFeed: async (author, url) => {
```

Comment:
Why this change?  It's not wrong, but it seems unrelated to this fix.

Reply:
@humphd When we doing the test, we found weird error that the name property got passed into the post object. After we changed this name to match the property name inside the post object, the error disappeared.

Reply:
If that's true, we should probably file a bug with ioredis or something.  For now, leave it as `author`, but please show me when I see you next time.

Reply:
Sure, will show you next week.

Reply:
Here is the error if we keep using "name"
![image](https://user-images.githubusercontent.com/47531873/69569060-d87a0400-0f8a-11ea-8bbb-08a1e9f1929e.png)

Reply:
I pulled your branch, and figured out the issue.  It's not `name` vs. `author`.  The problem is that you are re-using keys in Redis.  As I mentioned in my review, we need to use the `post.guid` as the key for a Post Hash, and to put into the sorted set.  You are using two auto-incrementing values, and the feedId and postId are colliding, causing the data to be missing in your tests.

If you implement the changes I requested, this should be fixed.